src/components/sbb-tooltip/sbb-tooltip.tsx

```diff
@@ -350,7 +350,7 @@ export class SbbTooltip extends LitElement {
         .filter((el) => el instanceof window.HTMLElement)
         .find((el) => (el as HTMLElement).matches(IS_FOCUSABLE_QUERY)) as HTMLElement;
       clearTimeout(this._closeTimeout);
-      await this.close();
+      this.close();
```

Comment:
If you remove that, function probably doesn't need to be async at all. Please leave a comment why await should not be used here (as e.b. I tend to always fixing not awaited calls ;) )

Reply:
oh I missed it :/
Nice catch

Reply:
Can you add a comment why it should not be awaited?

Reply:
It was needed because stencil forced the async on the `close()` method.

Reply:
Ah sorry, the close method is not async anymore